pytransform3d/test/test_batch_rotations.py

```diff
@@ -42,6 +43,34 @@ def test_norm_vectors_zero():
     assert_array_almost_equal(V_unit, V)
 
 
+def test_norm_axis_angles():
+    rng = np.random.default_rng(843)
+    A = np.hstack((
+        pbr.norm_vectors(rng.standard_normal(size=(10, 3))),
+        np.pi + rng.uniform(size=(10, 1))
+    ))
+
+    # 1D
+    assert_array_almost_equal(
+        pbr.norm_axis_angles(A[0]), pr.norm_axis_angle(A[0]))
+
+    assert_array_almost_equal(
+        pbr.norm_axis_angles(np.array([[[0.0, 0.0, 0.0, np.pi]]]))[0, 0],
+        pr.norm_axis_angle(np.array([0.0, 0.0, 0.0, np.pi])))
+
+    # 2D
+    A_norm = pbr.norm_axis_angles(A)
```

Comment:
Consider moving the definition of `A` down below, so it it closer to its usage - a reader does not have to scroll up to check what is in A ...

Reply:
It's used throughout the test case. I don't know how to solve it better. I don't want to make a new test case for each assert, since I would have to recreate the array in every function.